assets/xml/objects/object_driftice.xml

```diff
@@ -5,7 +5,7 @@
         <Texture Name="object_driftice_TLUT_000420" OutName="tlut_000420" Format="rgba16" Width="4" Height="4" Offset="0x420" />
         <Texture Name="object_driftice_Tex_000440" OutName="tex_000440" Format="ci4" Width="64" Height="64" Offset="0x440" />
         <Texture Name="object_driftice_Tex_000C40" OutName="tex_000C40" Format="ci4" Width="64" Height="64" Offset="0xC40" />
-        <DList Name="object_driftice_DL_0016A0" Offset="0x16A0" />
-        <Collision Name="object_driftice_Colheader_001AA8" Offset="0x1AA8" />
+        <DList Name="object_driftice_DL_0016A0" Offset="0x16A0" /> <!-- Original name is "obj_iceblock_model" -->
+        <Collision Name="object_driftice_Colheader_001AA8" Offset="0x1AA8" /> <!-- Original name is "obj_iceblock_bgdatainfo" -->
```

Comment:
Wait are you sure? there is an actual `obj_iceblock` object which is separate, is this a duplicate?

Reply:
Yes, I'm sure. `object_ice_block` still exists in MM3D, and has assets like this:
![image](https://user-images.githubusercontent.com/12245827/207515924-5ddffc15-f72d-41ce-8b20-bb2ba5f0b80b.png)